microshift_release_notes/microshift-4-14-release-notes.adoc

```diff
@@ -176,12 +213,10 @@ Red Hat Customer Portal user accounts must have systems registered and consuming
 This section will continue to be updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {product-title} {ocp-version}. Versioned asynchronous releases, for example with the form {product-title} {ocp-version}.z, will be detailed in subsections. In addition, releases in which the errata text cannot fit in the space provided by the advisory will be detailed in subsections that follow.
```

Comment:
```suggestion
This section is updated over time to provide notes on enhancements and bug fixes for future asynchronous errata releases of {product-title} {ocp-version}. Versioned asynchronous releases, for example with the form {product-title} {ocp-version}.z, are detailed in following subsections. In addition, releases in which the errata text cannot fit in the space provided by the advisory are detailed.
```

Reply:
I wonder if we should actually remove that last sentence. WDYT?

Reply:
Why not? I am not as familiar with that as you are. Doesn't seem to happen much, and not at all for MicroShift.